app/controllers/gigs.js

```diff
@@ -76,7 +76,7 @@ module.exports = function(app) {
         title: req.body.title,
         details: req.body.details,
         credits: parseInt(req.body.credits),
-        admin_task: req.body.admin_gig
+        admin_task: req.body.admin_gig ? true : false
```

Comment:
Checked again, something like this is needed because of the way forms are submitted. I changed it to `Boolean(req.body.admin_gig)` to be a little bit more clear what this is doing.